test/integration/testenv/happy-life-bank/seed.yml

```diff
@@ -51,6 +51,7 @@ accounts:
     id: 60257507-3191-4507-9d77-9071fd6b3c30
     initialBalance: 150000
     assetCode: MXN
+rates_url: 'http://host.docker.internal:8889/rates'
```

Comment:
I think this (and the one for cloud-nine) can go too, right? 
(along with `rates_url` in the `SeedInstance` interface of the mase-lib)